src/convertToObject.js

```diff
@@ -12,7 +12,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleMap = sourceString.split('\n').map(el => el.trim()).filter(el =>
```

Comment:
```suggestion
  const styleMap = sourceString
  .split('\n')
  .map(el => el.trim())
  .filter(el =>
```
Давай каждый метод писать с новой строки, а вот его начинку по возможности не переносить. Будет выглядеть читаемей

Reply:
Понял)